benchmarking/streamlit/app.py

```diff
@@ -92,11 +91,7 @@ def main() -> None:
                 st.session_state.llm_api = 'sncloud'
             else:  # SambaStudio
                 additional_env_vars = [
-                    'SAMBASTUDIO_BASE_URL',
-                    'SAMBASTUDIO_BASE_URI',
-                    'SAMBASTUDIO_PROJECT_ID',
-                    'SAMBASTUDIO_ENDPOINT_ID',
-                    'SAMBASTUDIO_API_KEY',
+                    'SAMBASTUDIO_URL',
```

Comment:
SAMBASTUDIO_API_KEY is not also an extra required env var?

Reply:
@snova-jorgep @snova-rodrigom is there a resolution to this? Thanks

Reply:
hey @snova-kwasia , yes we've talked about this and I'm working on it. I'll let you know once it's ready

Reply:
Alright, `env_input_fields` has been changed in order to be integrated to this kit. This is not affecting other dependent kits, I've tested EKR and Search Assistant to make sure about it. If you're good with it we can resolve this comment, thanks guys!